plugins/telemetry/telemetry_impl.cpp

```diff
@@ -391,7 +391,7 @@ void TelemetryImpl::process_mount_orientation(const mavlink_message_t &message)
     Telemetry::EulerAngle euler_angle {
         mount_orientation.roll,
         mount_orientation.pitch,
-        mount_orientation.yaw
+        mount_orientation.yaw_absolute
```

Comment:
This (http://mavlink.org/messages/common#MOUNT_ORIENTATION) has both `yaw` & `yaw_absolute`. Does the link hold message definitions of MAVLink2 ?
Also, if you change member here, we need to change its usage; isn't it ? :)

Reply:
Where do we need to change its usage? I'm confused.

Reply:
Ah OK! So, its just that [yaw_deg](https://github.com/dronecore/DroneCore/blob/742537418b5892bd30df3d58708826760c3dbef0/plugins/telemetry/telemetry.h#L74) of `EulerAngle` will be absolute now ?